Testscripts/Windows/VERIFY-INFINIBAND-MultiVM.ps1

```diff
@@ -317,6 +320,7 @@ function Main {
 				New-Item -Path "$LogDir\InfiniBand-Verification-$Iteration-$TempName" -Force -ItemType Directory | Out-Null
 				Move-Item -Path "$LogDir\$InfinibandNic-status*" -Destination "$LogDir\InfiniBand-Verification-$Iteration-$TempName" | Out-Null
 				Move-Item -Path "$LogDir\IMB*" -Destination "$LogDir\InfiniBand-Verification-$Iteration-$TempName" | Out-Null
+				Move-Item -Path "$LogDir\OSU*" -Destination "$LogDir\InfiniBand-Verification-$Iteration-$TempName" | Out-Null
```

Comment:
to make the naming consistent - OMB (which is the official name) might be a better alternative here.
(Same suggestion in other places as well.)

Reply:
Modified the name from OSU to OMB throughout.